nodes/srl/srl.go

```diff
@@ -167,10 +167,11 @@ func (s *srl) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 
 	// force cert generation for SR Linux nodes
 	if s.Cfg.Certificate == nil {
-		s.Cfg.Certificate = &types.CertificateConfig{
-			Issue: true,
-		}
+		// if certificate config deos not exist, create it
+		s.Cfg.Certificate = &types.CertificateConfig{}
 	}
+	// et issue to true, to finally force cert creation
+	s.Cfg.Certificate.Issue = true
```

Comment:
why changing this?

Reply:
so my thinking was, that we need to force and overrule the user if he "stupidly" decides to not generate a cert for an srl node to not create a certificat.
so if the user set 
```
certificate:
  issue: false
``` 
The s.Cfg.Certificate would not be nil, so we would ingest the possible `issue: false` value. Hence the existence of the struct `== nil` is decoupled from force setting the value to true.

Reply:
the struct is already initialized based on the Topology methods, so we just needed to set the field to *bool(true)